.storybook/preview.js

```diff
@@ -17,7 +17,7 @@ export const parameters = {
   docs: {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33126&commitId=e0640bed786dc083762f016768f7a74d03e98412&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|284| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>